cli-toolchain/cognito-triggers.md

```diff
@@ -0,0 +1,508 @@
+# Cognito Lambda Triggers
+
+Certain AWS Services can [invoke Lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html) in response to lifecycle events. The Amplify CLI provides trigger templates for common use cases.
+
+If you wish to modify the functionality of these templates, you are able to do so locally before pushing them.
```

Comment:
Should we say what directory they're in? Also should we note the "wrapper" function?

Reply:
I'm noting the directory.  The 'wrapper' function gets explained below.